<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data_table</class>
 <widget class="QDialog" name="Data_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>49</y>
     <width>780</width>
     <height>411</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>780</width>
     <height>420</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>463</y>
     <width>169</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nhãn Hiệu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nhiệt Luyện</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Kích Thước S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Độ Rắn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Giới hạn bền</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Giới hạn chảy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="Nhan_Hieu">
        <property name="styleSheet">
         <string notr="true">background:  lightgrey</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Nhiet_luyen">
        <property name="styleSheet">
         <string notr="true">background:  lightgrey</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Kich_Thuoc">
        <property name="styleSheet">
         <string notr="true">background:  lightgrey</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Gioi_han_chay_3">
        <property name="styleSheet">
         <string notr="true">background:  lightgrey</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Do_Ran_2">
        <property name="styleSheet">
         <string notr="true">background:  lightgrey</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Gioi_han_ben_3">
        <property name="styleSheet">
         <string notr="true">background:  lightgrey</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>620</y>
     <width>168</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
       </font>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="Do_Ran">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>560</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Độ rắn...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kich_thuoc">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>kích thước S...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Gioi_han_chay_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>620</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Giới hạn chảy...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Gioi_han_ben_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>590</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Giới hạn bền...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Công thức ứng suất tiếp xúc tương ứng :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Cong_thuc_tuong_ung">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>470</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:  lightgrey</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>41</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qml.qrc">
     <normaloff>:/dowload/Anh_cua_app+do_an/Search.png</normaloff>:/dowload/Anh_cua_app+do_an/Search.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>22</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>21</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>781</x>
     <y>50</y>
     <width>42</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="ADD_button">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">  #ADD_button{  border-width: 9px;
    border-radius: 15px;
    border-color:rgba(0,0,0,0.5); ;
    font: bold 14px;
    padding: 1px}
#ADD_button:hover{
background-color:	rgba(116, 114, 157, 0.2);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qml.qrc">
        <normaloff>:/dowload/Anh_cua_app+do_an/Add.png</normaloff>:/dowload/Anh_cua_app+do_an/Add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="EDIT_Button">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">  #EDIT_Button{  border-width: 9px;
    border-radius: 15px;
    border-color:rgba(0,0,0,0.5); ;
    font: bold 14px;
    padding: 1px}
#EDIT_Button:hover{
background-color:	rgba(116, 114, 157, 0.2);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qml.qrc">
        <normaloff>:/dowload/Anh_cua_app+do_an/Edit.png</normaloff>:/dowload/Anh_cua_app+do_an/Edit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DELETE_button">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">  #DELETE_button{  border-width: 9px;
    border-radius: 15px;
    border-color:rgba(0,0,0,0.5); ;
    font: bold 14px;
    padding: 1px}
#DELETE_button:hover{
background-color:	rgba(116, 114, 157, 0.2);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qml.qrc">
        <normaloff>:/dowload/Anh_cua_app+do_an/Delete.png</normaloff>:/dowload/Anh_cua_app+do_an/Delete.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>741</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="maximumSize">
       <size>
        <width>631</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Tìm kiếm ......</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Nhãn hiệu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kích thước</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nhiệt luyện</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Độ rắn</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Giới hạn bền</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Giới hạn chảy</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>530</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>≤</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>560</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_16">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>560</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_18">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>590</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_20">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>590</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_21">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_22">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>620</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_23">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_24">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>620</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_25">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_26">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>≤</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="qml.qrc"/>
 </resources>
 <connections/>
</ui>
